examples/dialout-chatbot/bot_daily.py

```diff
@@ -0,0 +1,128 @@
+        f"Starting bot with room_url: {room_url}, token: {token}, callId: {callId}, callDomain: {callDomain}, phone_number: {phone_number}")
+    # diallin_settings are only needed if Daily's SIP URI is used
+    # If you are handling this via Twilio, Telnyx, set this to None
+    # and handle call-forwarding when on_dialin_ready fires.
+    async with aiohttp.ClientSession() as session:
+        diallin_settings = None
+
+        transport = DailyTransport(
+            room_url,
+            token,
+            "Chatbot",
+            DailyParams(
+                api_url=daily_api_url,
+                api_key=daily_api_key,
+                dialin_settings=diallin_settings,
+                audio_in_enabled=True,
+                audio_out_enabled=True,
+                camera_out_enabled=False,
+                vad_enabled=True,
+                vad_analyzer=SileroVADAnalyzer(),
+                transcription_enabled=True,
+            )
+        )
+
+        tts = ElevenLabsTTSService(
+            aiohttp_session=session,
+            api_key=os.getenv("ELEVENLABS_API_KEY", ""),
+            voice_id=os.getenv("ELEVENLABS_VOICE_ID", ""),
+        )
+
+        llm = OpenAILLMService(
+            api_key=os.getenv("OPENAI_API_KEY"),
+            model="gpt-4o"
+        )
+
+        messages = [
+            {
+                "role": "system",
+                "content": "You are Chatbot, a friendly, helpful robot. Your goal is to demonstrate your capabilities in a succinct way. Your output will be converted to audio so don't include special characters in your answers. Respond to what the user said in a creative and helpful way, but keep your responses brief. Start by saying 'Oh, hello! Who dares dial me at this hour?!'.",
+            },
+        ]
+
+        tma_in = LLMUserResponseAggregator(messages)
+        tma_out = LLMAssistantResponseAggregator(messages)
+
+        pipeline = Pipeline([
+            transport.input(),
+            tma_in,
+            llm,
+            tts,
+            transport.output(),
+            tma_out,
+        ])
+
+        task = PipelineTask(pipeline, PipelineParams(allow_interruptions=True))
+
+        @transport.event_handler("on_dialout_answered")
+        async def on_dialout_answered(transport, participant):
+            logger.info(
+                f"dialout answered for callID: {callId}, callDomain: {callDomain}"
+            )
+            transport.capture_participant_transcription(
+                participant["participantId"]
+            )
+            await task.queue_frames([LLMMessagesFrame(messages)])
+
+        @transport.event_handler("on_call_state_updated")
+        async def on_call_state_updated(transport, state):
+            logger.info(
+                f"call state updated for callID: state: {state}, callId: {callId}, callDomain: {callDomain}"
+            )
+            transport.start_dialout(
+                settings={
+                    "phoneNumber": phone_number,
```

Comment:
With the latest pipecat (to be released), you can pass in a `callerId`, which is the `id` of the phone number that will be used to dial the `phoneNumber`.

For exmaple, if you have multiple phone numbers on Daily.co. 
https://docs.daily.co/reference/rest-api/phone-numbers/purchased-phone-numbers (see the `id` in the 200 OK)

Reply:
It might make sense to adapt the example to have both 
`sipUri` or `phoneNumber` depending on what was passed into the bot runner.